bitrise.yml

```diff
@@ -1,33 +1,43 @@
 format_version: "13"
 default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 
+step_bundles:
+  setup:
+    steps:
+    - activate-ssh-key@4: { }
+    - git-clone@8: { }
+    - script:
+        title: Setup pinned Go version
+        inputs:
+        - content: asdf install
+
 workflows:
   test:
+    triggers:
+      pull_request:
+      - source_branch: "*"
     steps:
+    - bundle::setup: { }
     - git::https://github.com/bitrise-steplib/steps-check.git:
         inputs:
         - skip_step_yml_validation: "yes"
 
   create-release:
```

Comment:
Instead of creating a release here, we should delegate to the control center project just like with all other binary tools